tests/test_person.py

```diff
@@ -4,10 +4,11 @@
 from mock import Mock, patch
 
 
-class Test(unittest.TestCase):
+class TestPerson(unittest.TestCase):
 	
 	def setUp(self):
 		self.person = Person("Marvin","FELLOW","YES")
+		self.person.id_no = 2234
 
 	def test_instance_valriables(self):
 		self.assertNotEqual(self.person.name,"",msg = "Invalid name credentials.")	
```

Comment:
Obvious test. You just said person name is 'Marvin' then asserting if 'Marvin' is not equal to "", which obviously is true.